.travis.yml

```diff
@@ -22,13 +22,15 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
+dist: trusty
+
 addons:
-  postgresql: 9.4
+  postgresql: 9.6
```

Comment:
I found that postgresql v9.4 requires a specific configuration `postgresql_using='payload::jsonb',` that is not required on v9.6.
But it looks working. So, I'll downgrade to the version 9.4.
Now the only issue is mysql, with:
`AttributeError: 'function' object has no attribute '_compiler_dispatch'`